packages/events/src/util.ts

```diff
@@ -55,3 +56,47 @@ export function getEventServiceUrl(
     }
   });
 }
+
+/**
+ * Get the events query based on type.
+ * @param type - string to indicate event type with the options `upcoming`, `past`, `cancelled` and `draft`
+ * @param params - query features request options
+ * @returns an IQueryFeaturesRequestOptions that has the same values as `params` but for the modified Where and OrderBy properties
+ */
+export function getEventQueryFromType(
+  type: string,
```

Comment:
if you're already going back in here one more time...
```ts
// not just _any_ string
type: "upcoming" | "past" | "cancelled" | "draft"
```

Reply:
I did not know I could do that! that's awesome, thanks @jgravois